example/src/screens/Room.tsx

```diff
@@ -106,74 +113,25 @@ export const Room = () => {
           <View style={{ flex: 1 }}>
             {shouldShowParticipants && (
               <>
```

Comment:
We no longer need the outer fragment (`<>`)

Reply:
I don't think so. I tried removing that and it gives an error, which the editor constantly recommends to fix by wrapping in empty jsx tag.

Reply:
Done.